docs/API.md

```diff
@@ -149,10 +149,10 @@ back at vehicle end.
 Use `skills` to describe a problem where not all tasks can be served
 by all vehicles. Job skills are mandatory, i.e. a job can only be
 served by a vehicle that has **all** its required skills. In other
-words: job `j` is eligible to vehicle `v` iff `j.skills` is included
+words: job `j` is eligible to vehicle `v` if `j.skills` is included
 in `v.skills`.
 
-In order to ease modeling problems with no skills required, it is
+In order to ease modelling problems with no skills required, it is
```

Comment:
Sounds like yet another US vs rest of the world difference: https://grammarist.com/spelling/model/. Maybe we should simply go with "describing"?